bedrock/firefox/views.py

```diff
@@ -442,15 +417,15 @@ def get_template_names(self):
                 template = 'firefox/dev-whatsnew.html'
         elif channel == 'nightly':
             template = 'firefox/nightly_whatsnew.html'
-        elif show_57_whatsnew(version):
+        elif version.startswith('57.'):
```

Comment:
Simplified the logic here to sandbox 56 & 57 `/whatsnew` pages, since we don't want to see them again for future releases.